modules/global/src/com/haulmont/cuba/core/global/View.java

```diff
@@ -49,16 +49,21 @@
      * Parameters object to be used in constructors.
      */
     public static class ViewParams {
-        protected View src;
+        protected List<View> src;

```

Comment:
Set to Collections.emptyList() by default thus there will be no need to check for null